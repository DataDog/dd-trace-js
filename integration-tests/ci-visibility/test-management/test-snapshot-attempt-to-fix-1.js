'use strict'

let retryCounter = 0

describe('attempt to fix snapshot', () => {
  it('is flaky', () => {
    if (process.env.SHOULD_PASS_ALWAYS) {
      expect(3).toMatchSnapshot()
    } else {
      const sum = ++retryCounter > 2 ? 3 : 4
      expect(sum).toMatchSnapshot()
    }
    expect('a').toMatchSnapshot()
  })
})
