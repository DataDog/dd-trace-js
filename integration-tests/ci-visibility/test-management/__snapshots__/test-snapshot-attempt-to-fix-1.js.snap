// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`attempt to fix snapshot is flaky 1`] = `3`
exports[`attempt to fix snapshot is flaky 2`] = `"a"`
